tests/migrations/0006_basetestapplication_token_family.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 5.2 on 2024-08-09 16:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tests', '0005_basetestapplication_allowed_origins_and_more'),
```

Comment:
needs `migrations.swappable_dependency`?

Reply:
Added it. But I'm not quite sure if it is needed. In contrast to `0001_initial.py`, the migration itself is not directly referring to the model, just to the model name with a string. In `0001_initial.py`, the setting `oauth2_settings.REFRESH_TOKEN_MODEL` and `settings.AUTH_USER_MODEL` are used directly (although only for AUTH_USER_MODEL the dependency is declared, but maybe it's a mistake from the past that the REFRESH_TOKEN_MODEL wasn't declared?).

So I'm wondering what the option would actually do.

Reply:
I asked a possible liar (ChatGPT): 

> Yes, when working with swappable models in Django, you should add migrations.swappable_dependency() to the auto-generated migration code. This ensures that the migration system correctly handles dependencies between the swappable models and other models.
> 
> What is migrations.swappable_dependency()?
> The migrations.swappable_dependency() function is used to specify a dependency on a swappable model. This ensures that migrations are applied in the correct order, especially when custom models are used to replace the default ones.